### PR TITLE
GOV-290 Enabled TLS for tomcat server

### PR DESCRIPTION
**Pull request by danishjamal104**
## Description

* Implemented GOV-290. Enabled TLS for tomcat server.
* Older minor fix for mojaloop.

Please refer [this](https://mifosforge.jira.com/browse/GOV-290) ticket for more info.

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Design related bullet points or design document link related to this PR added in the description above. 

- [x] Updated corresponding Postman Collection or Api documentation for the changes in this PR.

- [x] Create/update unit or integration tests for verifying the changes made.

- [x] Add required Swagger annotation and update API documentation with details of any API changes if applicable

- [x] Followed the naming conventions as given in https://docs.google.com/document/d/1Q4vaMSzrTxxh9TS0RILuNkSkYCxotuYk1Xe0CMIkkCU/edit?usp=sharing

### CODE REVIEW

File: src/main/resources/application.yml [hunk 1]
```diff
@@ -20,6 +20,12 @@ mpesa:
 timer: "PT45S"
 
 server:
+  ssl:
+    key-alias: "tomcat-https"
+    key-store: "classpath:keystore.jks"
```
Comment (fynmanoj) [inline src/main/resources/application.yml:25]: can this be an external location maintained by DevOps?

File: src/main/java/org/mifos/connector/channel/zeebe/ZeebeWorkers.java [hunk 2]
```diff
@@ -74,7 +74,12 @@ private void workerSendErrorToChannel(){
                                 errorInformation.getString("errorDescription"));
                     }
 
+                    Map<String, Object> existingVariables = job.getVariablesAsMap();
+
+                    existingVariables.put("transferCreateFailed", "false");
```
Comment (fynmanoj) [inline src/main/java/org/mifos/connector/channel/zeebe/ZeebeWorkers.java:79]: create a String constant for the literal and use

**Review by fynmanoj — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yml (hunk 1 above)
Comment (danishjamal104) [reply src/main/resources/application.yml:25]: no idea.... need to read kubernetes doc.

File: src/main/java/org/mifos/connector/channel/zeebe/ZeebeWorkers.java (hunk 2 above)
Comment (danishjamal104) [reply src/main/java/org/mifos/connector/channel/zeebe/ZeebeWorkers.java]: DONE

**Pull request merged**